src/Reflection/Adapter/ReflectionParameter.php

```diff
@@ -185,19 +174,16 @@ public function getDefaultValueConstantName()
         return $this->betterReflectionParameter->getDefaultValueConstantName();
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    public function hasType()
+    public function hasType(): bool
     {
         return $this->betterReflectionParameter->hasType();
     }
 
     /**
-     * {@inheritDoc}
+     * @psalm-suppress MismatchingDocblockReturnType
```

Comment:
I suppose this is because the parent class returns `ReflectionType`?

Reply:
Yes.